easybuild/easyconfigs/p/python-pulp/python-pulp-2.5.1-foss-2021a.eb

```
@@ -0,0 +1,35 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'PythonPackage'
+
+name = 'python-pulp'
```

Based on its README file, the name of this software should be PuLP. We only prepend software names with *python* in cases where the raw name is already in use. This usually happens with python bindings to other software.
```suggestion
name = 'PuLP'
```

---

> Based on its README file, the name of this software should be PuLP. We only prepend software names with _python_ in cases where the raw name is already in use. This usually happens with python bindings to other software.

@lexming `python` packages are many and clog the directory with simple `python` packages are difficult to visualize what packages are still required for maintenance. A prefix, I think would be a good idea, i.e. `spack` adopts such a convention.